app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  validates :name, :photo, :bio, presence: true
+  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
+  attr_accessor :photo, :bio, :email
```

Comment:
- [ ] Well done for creating the association between user and posts 👏. Please also do the same for:
1. user and comments
2. user and likes